classes/05class/exercises/c05-make01/deniseddi/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM httpd:2.4.41-alpine
+COPY index.html /usr/local/apache2/htdocs/
+# EXPOSE 80
```

Comment:
Hi Denise, why is this line commented out? You can leave it, it works as documentation for your docker file

>The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published.